mongodbatlas/resource_mongodbatlas_third_party_integration_test.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
 func TestAccResourceMongoDBAtlasThirdPartyIntegration_basic(t *testing.T) {
+	t.Skip() // TODO: Address failures in API returning obfuscated values
```

Comment:
it would be great to add a CLOUDP link to this comment to track when this change will be done